src/test/java/org/simlar/simlarserver/services/pushnotification/ApplePushNotificationMockServerTest.java

```diff
@@ -50,18 +49,21 @@ private void createMockServerRequest(final String deviceToken, final String expi
                 ).respond(response);
     }
 
+    private String requestVoipPushNotification(final String deviceToken, final Instant expiration) {
```

Comment:
Okay, here `applePushNotification` and `mockServer.getLocalPort()` are both preventing  `requestVoipPushNotification` from beng static...